lib/travis/api/v3/models/env_var.rb

```diff
@@ -0,0 +1,18 @@
+module Travis::API::V3
+  class Models::EnvVar < Travis::Settings::Model
+    attribute :id, Integer
+    attribute :name, String
+    attribute :value, String
+    attribute :public, Boolean
+    attribute :repository_id, Integer
+
+    def repository
+      @repository ||= Models::Repository.find(repository_id)
```

Comment:
I think we should be using 
```ruby
Models::Repository.find_by(id: repository_id)
```
and handling the nil case gracefully as I couldn't find where the exception would be handled if it can't find a repo w/ that id, but I'm not 100% sure we aren't handling it somewhere. However, we might be giving a mis-leading message back to the user.